util/scripts/install/install.sh

```diff
@@ -12,18 +12,25 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
+mkdir logs
+
 # backend
-util/scripts/install/backend.sh
+util/scripts/install/backend.sh > logs/backend.log
+
+# nomos scanner for dosocs
+util/scripts/install/nomos.sh > logs/nomos.log 2>logs/nomos.err 
```

Comment:
`nomos` does not get installed correctly using this script on my machine. I get [this](https://pastebin.com/qSJKKwpX) following stack trace. I'm running OSX 10.14.6 and I'm using the `fish` shell version 3.0.2.

Reply:
Yeah, right now its a shit show on the Mac. I am working through those issues. I want to include the script in dev without calling it automatically in order to enable deployment of augur-sbom on Ubuntu servers.

Reply:
Pushing it to master will help us get others involved in evaluating the new functionality. Because its experimental until we resolve the mac issues, is not automatically installed, and increasingly vital on Ubuntu, I would like to push this forward.

Reply:
FWIW, I've never been able to get `nomos` installed on macOS. Given, I did try almost a year ago, but the error I got now does look very similar to the one I got then. It makes me extremely uneasy to push something that doesn't work to `master` but since it's non-mission-critical (well at least probably not, for most users) and this PR contains a lot of other risk work, I'm willing to put aside my fears for now. 

That being said, IMO our number one focus after the call today ought to be either getting `nomos` to install on macOS, or to make the install optional and put in a caveat that it won't work on macOS, at least for the time being.